### ISSUE TITLE
[Future]: Towny support

### ISSUE THREAD

**Issue by ErythroCraft** [smaks6/Nokaut#3]
Hi. 

I've seen your plugin and I think it is a nice idea.
But there is towny support missing to enable or disable in wild.

1. Can you give support for Towny?
  Towny: true/false
  onlyWild: true/false

Plus permission like LP

**Comment by smaks6** [comment]
You mean https://www.spigotmc.org/resources/towny-advanced.72694/?

**Comment by ErythroCraft** [comment]
Yes. Your plugin would better working with a different of wild, towns, nations and plots.
